tasks/main.yml

```diff
@@ -5,10 +5,13 @@
     owner: root
     group: root
     state: directory
+  become: true
```

Comment:
Good point, I was doing that in the playbook before. Better here for re-use I guess.